spec/index.bs

```diff
@@ -1113,6 +1101,20 @@ Given an {{HTMLScriptElement}} (|script|), this algorithm performs the following
 
     If the algorithm threw an error, rethrow the error.
 
+## <dfn-abstract-op export>Validate attribute mutation</dfn> ## {#validate-attribute-mutation}
+Given {{Attr}} |attribute|, {{Element}} |element| and {{TrustedType}} or a string |newValue|, this algorithm performs the following steps:
+
+1. If |element| has an <a spec="webidl" lt="attribute">IDL attribute</a> that [=reflects=] a content attribute with |attribute|'s [=Attr/local name=], let |idlAttribute| be that IDL attribute. Otherwise, return stringified |newValue|.
+1. If {{StringContext}} IDL extended attribute does not appears on |idlAttribute|, return stringified |newValue|.
+1. Set |value| to the result of executing [$Get Trusted Type compliant string$] with the following arguments:
+     * {{StringContext}} identifier as |expectedType|
+     * |newValue| as |input|
+     * |element|'s <a>relevant global object</a> as |global|
+     * the result of [=concatenating=] the list &laquo; |element|'s [=Element/local name=], |attribute|'s |localName| &raquo; with `"."` as a |separator| as |sink|
```

Comment:
Nit: Use the code points directly rather than entities. Makes it easier to see what is going on. Also no need to prefix with "the list".